portfolio/src/main/webapp/script.js

```diff
@@ -33,12 +33,17 @@ function openLink(link){
 
 function getHello() {
   //Concatenates each comment to display on page
-  fetch('/data').then(response => response.json()).then((comments) => {
-    var comments_text = '';
-    for (index in comments.comments) {
-      comments_text = comments_text.concat(comments.comments[index], '\n');
-    }
-    document.getElementById("greet-message").innerText = comments_text;
-    
+  fetch('/data').then(response => response.json()).then((jsonComments) => {
+    const commentList = document.getElementById("greet-message");
+    jsonComments.comments.forEach((comment) => {
+      commentList.append(createListElement(comment.value));
```

Comment:
It might make more sense to use `appendChild` and have `commentList` be an unordered or ordered list. Right now it looks like `document.getElementById("greet-message")` is just a div.